### PR TITLE
add FAQ about kafka to email

### PR DESCRIPTION
**Pull request by jracusin**

### CODE REVIEW

**Comment by jracusin** [comment]
> Does this really go on the FAQ? How about the roadmap?

I got a question from someone specifically about this, so I thought it would be good to add it.  Once we implement this, we can remove it from FAQ.  It should probably also be on roadmap.  I'll add it.

**Comment by lpsinger** [comment]
> It should probably also be on roadmap. I'll add it.

In this PR?

**Comment by jracusin** [comment]
I just added info on GCN over email to the roadmap, also as part of this PR.

File: app/routes/docs/roadmap.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ meta:
 
 The following features are coming soon to GCN.
 
+## GCN Notices over Email
+
+Users can now stream GCN Notices with GCN Classic over Kafka using the python or javascript source code generated via the [QuickStart guide](https://gcn.nasa.gov/quickstart). Self-subscription of GCN Notices delivered via email will available in a near-future release.
```
Comment (lpsinger) [inline app/routes/docs/roadmap.md:12]:
```suggestion
Users can now stream GCN Notices with GCN Classic over Kafka using the Python or JavaScript source code generated via the [Start Streaming GCN Notices](https://gcn.nasa.gov/quickstart) quick start guide. Self-subscription of GCN Notices delivered via email will available in a near-future release.
```

File: app/routes/docs/faq.md [hunk 2]
```diff
@@ -20,3 +20,7 @@ See [Client Configuration](docs/client).
 ## What does the warning `Subscribed topic not available: gcn.classic.text.AGILE_GRB_GROUND: Broker: Unknown topic or partition'` mean?
 
 This warning means that there have not been any recent alerts on that topic.
+
+## How do I receive GCN Notices via email from GCN Classic over Kafka?
+
+This feature is under development and will be released in the near future. Currently GCN Classic over Kafka serves GCN notices only via the Kafka broker, which you can setup via the [QuickStart guide](https://gcn.nasa.gov/quickstart).
```
Comment (lpsinger) [inline app/routes/docs/faq.md:26]:
```suggestion
This feature is under development and will be released in the near future. Currently GCN Classic over Kafka serves GCN notices only via the Kafka broker, which you can setup via the [Start Streaming GCN Notices](https://gcn.nasa.gov/quickstart) quick start guide.
```

(We don't every really call it the QuickStart guide.)

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]

**Pull request merged**